scanner/mapping.go

```diff
@@ -70,10 +71,17 @@ func (s mediaFileMapper) toMediaFile(md metadata.Tags) model.MediaFile {
 	mf.MbzAlbumType = md.MbzAlbumType()
 	mf.MbzAlbumComment = md.MbzAlbumComment()
 	mf.Comment = s.policy.Sanitize(md.Comment())
-	mf.Lyrics = s.policy.Sanitize(md.Lyrics())
 	mf.Bpm = md.Bpm()
 	mf.CreatedAt = time.Now()
 	mf.UpdatedAt = md.ModificationTime()
+	lyrics := s.policy.Sanitize(md.Lyrics())
+	if isSynced(lyrics) {
+		mf.SyncedLyrics = lyrics
+		mf.UnsyncedLyrics = ""
+	} else {
+		mf.UnsyncedLyrics = lyrics
+		mf.SyncedLyrics = ""
+	}
```

Comment:
```suggestion
	mf.SyncedLyrics = ""
	mf.UnsyncedLyrics = ""
	if isSynced(lyrics) {
		mf.SyncedLyrics = lyrics
	} else {
		mf.UnsyncedLyrics = lyrics
	}
```

Reply:
Alternative: unless there is any reason to store both synced and unsynced lyrics, it makes more sense to have a single `lyrics` field and a boolean called `is_lyrics_synced`.

Reply:
@KartikSoneji: Yes, there is a reason: Currently we only import one lyrics tag, but as `taglib` supports multi-valued tags, we can import both versions. This will require a small change in the `metadata.Tags.Lyrics()` method, for it to return an array, but can be implemented in a follow-up step

Reply:
Right, but is there any reason to keep unsynced lyrics if the synced version is available?

Reply:
Yes, Subsonic clients expect unsynced lyrics. Unless we remove the sync markup dynamically, we need both. Maybe that's actually a good idea ...

Reply:
https://github.com/navidrome/navidrome/blob/c635fdde6f7fcec2d2a97a27eb739c2ece7fd448/scanner/mapping.go#L178
Yes, we should be able to use the same regex to remove the timestamps from synced lyrics.

Reply:
I think we don't even need the `is_lyrics_synced` field, as we can detect it at runtime using that regex. Hey @dheerajdlalwani , do you think we can change this? To have one `lyrics` field only? We wouldn't need the migration in this case. Ping me on Discord if you have any questions